src/services/base.service.js

```diff
@@ -91,6 +91,7 @@ const baseCreate = async ({ req }, resourceType) => {
   checkSupportedResource(data.resourceType);
   //Create a new id regardless of whether one is passed
   data['id'] = uuidv4();
+  data['meta'] = { lastUpdated: getCurrentInstant(), ...data['meta'] };
```

Comment:
Unfortunately this causes a bug where, if `data.meta.lastUpdated` is provided in the request body, the `getCurrentInstant` value gets overwritten since the `...data['meta']` appears later in this object creation.

I'd switch the order of these two object properties and ensure that it overwrites any value that might be provided in the request body and add a comment explaining why the order matters.

I'd also recommend adding a unit test case for this overwriting functionality.